frontend/src/scenes/onboarding/sdks/product-analytics/android.tsx

```diff
@@ -16,29 +17,36 @@ function AndroidCaptureSnippet(): JSX.Element {
 
 function AdvertiseAndroidReplay(): JSX.Element {
     return (
-        <div>
-            <h3 className="mt-8">
-                Session Replay for Android <LemonTag type="highlight">NEW</LemonTag>
-            </h3>
-            <div>
-                Session replay is now in beta for Android.{' '}
-                <Link to={urls.onboarding('session_replay', OnboardingStepKey.INSTALL, SDKKey.ANDROID)}>
-                    Learn how to set it up
-                </Link>
-            </div>
-        </div>
+        <>
+            <LemonBanner type="info" className="mt-8">
+                <div>
+                    <h3>
+                        Session Replay for Android <LemonTag type="highlight">NEW</LemonTag>
+                    </h3>
+                    <div>
+                        Session replay is now in beta for Android.{' '}
+                        <Link
+                            to={urls.onboarding('session_replay', OnboardingStepKey.INSTALL, SDKKey.ANDROID)}
+                            data-attr="product-analytics-onboarding-android-replay-cta"
+                        >
+                            Learn how to set it up
+                        </Link>
+                    </div>
+                </div>
+            </LemonBanner>
+        </>
```

Comment:
Not blocking. The wrapping div might also be removable

```suggestion
        <LemonBanner type="info" className="mt-8">
            <div>
                <h3>
                    Session Replay for Android <LemonTag type="highlight">NEW</LemonTag>
                </h3>
                <div>
                    Session replay is now in beta for Android.{' '}
                    <Link
                        to={urls.onboarding('session_replay', OnboardingStepKey.INSTALL, SDKKey.ANDROID)}
                        data-attr="product-analytics-onboarding-android-replay-cta"
                    >
                        Learn how to set it up
                    </Link>
                </div>
            </div>
        </LemonBanner>
```